### PR TITLE
Address playback

### PR DESCRIPTION
**Pull request by dcdarrell9**
### What is the context of this PR?
- Playback address from metadata on separate lines instead of on one.

### How to review 
- Check that address comes gets displayed as it should.
- Not sure how the address should display in a summary, so possibly need some guidance on that

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: app/jinja_filters.py
Comment (insacuri) [inline app/jinja_filters.py]: Addresses instead of address'

File: app/jinja_filters.py
Comment (insacuri) [inline app/jinja_filters.py]: I think instead of address1 and address2, it should be something like `user_entered_address` and `metadata_address` maybe?  Just something more descriptive.

File: app/jinja_filters.py
Comment (insacuri) [inline app/jinja_filters.py]: This isn't a date is it?  Is this a copy/pasta error? ^^
(Same comment with the line below)

**Review by insacuri — CHANGES_REQUESTED** [review]

File: app/jinja_filters.py
Comment (insacuri) [inline app/jinja_filters.py]: Still says date

File: tests/app/test_jinja_filters.py
Comment (jonnyshaw89) [inline tests/app/test_jinja_filters.py]: Can you create a test where you have missing values from both that don't match?
eg 
```
address1 = ['44', None, 'Gardens', None, 'SA1 1AA']
address2 = [None, 'Testy', None, 'Newport', None]
```

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: tests/app/test_jinja_filters.py
Comment (jonnyshaw89) [reply tests/app/test_jinja_filters.py]: Or would the `None`s be `''`?

File: app/jinja_filters.py
Comment (dcdarrell9) [reply app/jinja_filters.py]: I'm so sure I changed that 🤣

File: tests/app/test_jinja_filters.py
Comment (dcdarrell9) [reply tests/app/test_jinja_filters.py]: Both come back as `''` unless all the answers the answers were `None` or `''` then an exception would be raised

**Comment by dcdarrell9** [comment]
If you leave the answers blank and submit then an exception is raised, we need to know whether we have an `at least one` of a certain amount of answers, so this probably needs some other type of testing/analysis/research or something along those lines to work out what the expected functionality is.

File: app/jinja_filters.py
Comment (jonnyshaw89) [inline app/jinja_filters.py]: I wonder if exception it a bit harsh?

File: app/jinja_filters.py
Comment (insacuri) [inline app/jinja_filters.py]: Sorry to be super picky, but could we rename `a` to `field` or something like that?  Makes it easier at a glance that it's a list of fields

File: app/jinja_filters.py
Comment (insacuri) [inline app/jinja_filters.py]: More pickiness sorry, but could you expand these a bit more and say that they're lists of address fields.  Makes it easier to figure out what's going on at a glance

File: app/jinja_filters.py
Comment (dcdarrell9) [reply app/jinja_filters.py]: That's what I'm thinking, if it's required to have at least one of these, then exception would be right, but if not then I'm not too sure, possible to redo this so that if the `user_entered_address` is empty then it defaults back to the `metadata_address` I think that way may be better?

Comment (jonnyshaw89) [reply app/jinja_filters.py]: You need to check the behaviour with Donna

**Review by bitdivision — COMMENTED** [review]
Looks good. Happy to approve once existing comments are addressed.

**Review by insacuri — APPROVED** [review]

File: app/jinja_filters.py
Comment (bitdivision) [reply app/jinja_filters.py]: Did you resolve this @dcdarrell9 ?

File: app/jinja_filters.py
Comment (bitdivision) [inline app/jinja_filters.py]: Could you use concatenated_list here to keep things consistent?

Comment (dcdarrell9) [reply app/jinja_filters.py]: Didn't see that, looks like it works, so gonna test a little 👍

Comment (bitdivision) [reply app/jinja_filters.py]: Not a big deal really, just a little nicer.

File: tests/app/test_jinja_filters.py
Comment (MebinAbraham) [inline tests/app/test_jinja_filters.py:882]: Not sure if this conversation happened, but is it really wrong to use metadata in this scenario?

Comment (bitdivision) [reply tests/app/test_jinja_filters.py:882]: I think that is probably what should happen, but needs to be confirmed with Donna.

**Comment by jonnyshaw89** [comment]
@dcdarrell9 still waiting on confirmation about this https://github.com/ONSdigital/eq-survey-runner/pull/1751#discussion_r214864482

**Comment by dcdarrell9** [comment]
@jonnyshaw89 There's a possibility of having 2 types of question...
1. At least one of need to be answered.
2. If none are answered, default to metadata

**Review by jonnyshaw89 — DISMISSED** [review]

File: app/jinja_filters.py
Comment (dwyeradam) [inline app/jinja_filters.py]: If you do find issues @jonnyshaw89 can you add that this `if and elif` combo should just be an `or` as it's doing the same thing.

Comment (dcdarrell9) [reply app/jinja_filters.py]: Yeah, I thought about that, but then thought it looks easier to read like this, happy to change

Comment (bitdivision) [reply app/jinja_filters.py]: Definitely better as an `or`. 
Otherwise you're using an if statement purely as a way to split up lines, which isn't what anyone would expect.

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**
